src/kv/committable_tx.h

```diff
@@ -23,8 +33,11 @@ namespace kv
 
     kv::TxHistory::RequestID req_id;
 
-    SerialisedEntryFlags flags = 0;
+  public:
```

Comment:
Should this be removed?

Reply:
I meant `public:`. It seems that these members are only accessed inside `CommittableTx`?